README.md

```diff
@@ -1,132 +1,88 @@
 
-## Table of Contents [if the explainer is longer than one printed page]
+## Proposed API
 
-[You can generate a Table of Contents for markdown documents using a tool like [doctoc](https://github.com/thlorenz/doctoc).]
+Apps that want to be shown on the lock screen would indicate this by declaring a URL in their manifest, to be used to launch the app on the lock screen. The URL must be within the _scope_ defined in the manifest. This URL could be a new manifest entry such as _lock\_screen\_start\_url_ but a more generic configuration would allow other custom launch actions to be added in future. We propose adding a _special\_launchers_ list to the manifest, in which items declare a _URL_ and a _purpose_ that may be recognised by user agents.
 
-<!-- START doctoc generated TOC please keep comment here to allow auto update -->
-<!-- END doctoc generated TOC please keep comment here to allow auto update -->
-
-## Introduction
-
-[The "executive summary" or "abstract".
-Explain in a few sentences what the goals of the project are,
-and a brief overview of how the solution works.
-This should be no more than 1-2 paragraphs.]
-
-## Goals [or Motivating Use Cases, or Scenarios]
-
-[What is the **end-user need** which this project aims to address?]
-
-## Non-goals
-
-[If there are "adjacent" goals which may appear to be in scope but aren't,
-enumerate them here. This section may be fleshed out as your design progresses and you encounter necessary technical and other trade-offs.]
-
-## [API 1]
-
-[For each related element of the proposed solution - be it an additional JS method, a new object, a new element, a new concept etc., create a section which briefly describes it.]
-
-```js
-// Provide example code - not IDL - demonstrating the design of the feature.
-
-// If this API can be used on its own to address a user need,
-// link it back to one of the scenarios in the goals section.
-
-// If you need to show how to get the feature set up
-// (initialized, or using permissions, etc.), include that too.
+```
+"lock_screen": {
+  "start_url": "/some/url" 
+}
 ```
 
-[Where necessary, provide links to longer explanations of the relevant pre-existing concepts and API.
-If there is no suitable external documentation, you might like to provide supplementary information as an appendix in this document, and provide an internal link where appropriate.]
-
-[If this is already specced, link to the relevant section of the spec.]
-
-[If spec work is in progress, link to the PR or draft of the spec.]
-
-## [API 2]
-
-[etc.]
-
-## Key scenarios
-
-[If there are a suite of interacting APIs, show how they work together to solve the key scenarios described.]
-
-### Scenario 1
-
-[Description of the end-user scenario]
+To use the API users would start by calling window.getLockScreenData().
 
 ```js
-// Sample code demonstrating how to use these APIs to address that scenario.
+const data = await window.getLockScreenData();
```

Comment:
IMO this example fits better if placed right after "The app may use this API to send data from the lock screen instance to the unlocked instance of the app and vice versa" in the paragraph below, or after that paragraph.